src/queries/queries.js

```diff
@@ -91,3 +95,11 @@ export const buildTimeseriesQueryByDateNoLimit = date => {
     }
   }
 }
+
+export const buildSettingsQuery = () => ({
+  definition: Q(SETTINGS_DOCTYPE),
+  options: {
+    as: SETTINGS_DOCTYPE,
+    fetchPolicy: CozyClient.fetchPolicies.olderThan(older30s)
```

Comment:
nit: I think queries are easy to test, and one test would document exactly why we choose a fetch policy of 30 seconds. That is why I recommend to test that part.